iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/CmdTreeImplTest.java

```diff
@@ -67,7 +86,7 @@ void shouldCheckParseCmdTree() {
   }
 
```

Comment:
You are missing test case for shell form, which would highlight a `NullPointerException` in `CmdTreeImpl` ;) 
```suggestion


  @Test
  void shouldCheckParseCmdShellFormTree() {
    CmdTree tree = DockerTestUtils.parse("CMD executable param1 param2", DockerLexicalGrammar.CMD);
    assertThat(tree.getKind()).isEqualTo(DockerTree.Kind.CMD);
    assertThat(tree.keyword().value()).isEqualTo("CMD");

    assertThat(tree.cmdArguments().stream().map(TextTree::value)).containsExactly("executable", "param1", "param2");
    assertThat(((SyntaxToken)tree.children().get(0)).value()).isEqualTo("CMD");

    ExecFormTree execForm = (ExecFormTree) tree.children().get(1);
    assertThat(execForm).isSameAs(tree.execForm());
  }
```

Reply:
Thank you for the test suggestion